README.md

```diff
@@ -13,6 +13,7 @@ Builders' and Angular **major** versions **must** match.
 <details>
   <summary>Click to expand</summary>
 
+- [Version 20](https://github.com/just-jeb/angular-builders/tree/20.x.x)
```

Comment:
```suggestion
- [Version 21](https://github.com/just-jeb/angular-builders/tree/21.x.x)
- [Version 20](https://github.com/just-jeb/angular-builders/tree/20.x.x)
```

Reply:
I usually create the previous version branch (in this case 20.x.x) moments before merging the new version branch.